packages/terra-hyperlink/src/Hyperlink.jsx

```diff
@@ -43,17 +43,25 @@ const propTypes = {
   /**
    * The content to display inside link.
    */
-  children: PropTypes.node,
+  text: PropTypes.string.isRequired,
```

Comment:
Please adjust the ordering so the props are in alphabetical order.

Reply:
Done [ace4b41](https://github.com/cerner/terra-core/pull/3605/commits/ace4b410994db19419f23f2538fa408ceb027904)